python/test/t_PointToFieldFunctionalChaosAlgorithm_std.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+import openturns.testing as ott
+
+ot.TESTPREAMBLE()
+# ot.Log.Show(ot.Log.INFO)
+
+# Time grid parameters
+T = 3.0
```

Comment:
Using a single upcase letter is against the PEP8:

https://peps.python.org/pep-0008/#names-to-avoid